app/components/autocomplete_selector/autocomplete_selector.tsx

```diff
@@ -47,7 +47,7 @@ type Props = {
 
 type State = {
     selectedText: string;
-    selected?: DialogOption | DialogOption[];
+    selected?: DialogOption[];
```

Comment:
Always accept an array of DialogOtions.

Reply:
This makes things simpler on our code, but does it make things simpler for the app developer? I mean... the general case will be something like `selected: [{label: "someLabel", value:"someValue"}]`. Is the value this change adds worth the extra `[]` on the developer experience.

Reply:
After thinking about it, I agree it makes the code simpler, but having the developer add stuff one object in an array isn't worth it.  

I reverted this change. Please re-review again. I added the multiselect handling.